db/migrate/20181205201208_add_allow_doing_subsequent_review_without_first_round_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddAllowDoingSubsequentReviewWithoutFirstRoundToAssignments < ActiveRecord::Migration
+  def change
+    add_column :assignments, :allow_doing_subsequent_review_without_first_round, :boolean
```

Comment:
Professor suggests change `allow doing subsequent review without first round` to `allow selecting additional reviews after 1st round`.